CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+### Changed
+
+- Units.unit - `units.unit.ns` now returns `1/4` within an open `qm.qua.program` scope and `1` otherwise. Same for `us`, `ms`, `s`, and `clock_cycle`. By default results of `a * ns` operation are cast to `int` and a warning is generated if casting discards a nonzero remainder.
+
+  ```python
+  unit.ns == 1 # Times are in ns in python
+
+  with program() as prog:
+    4 * unit.ns == 1 # Times are in clock cycles in qua
+  ```
+
```

Comment:
This part should be removed from the changelog and should only be in the readme. The changelog part above is great!

Reply:
Thanks! 
Done! :)